utilities.iss

```diff
@@ -71,6 +71,65 @@ begin
   end;
 end;
 
+// Gets the device contextt
+function GetDC(HWND: DWord): DWord; external 'GetDC@user32.dll stdcall';
```

Comment:
Needs testing on WINE probably

Reply:
I'm assuming it'll work fine since Wine should correctly translate the API call. Of course it should still be tested.